src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MyExtensionsApp._1.csproj

```
@@ -270,7 +270,7 @@
         <Compile Update="**\*.xaml.cs">
           <DependentUpon>%(Filename)</DependentUpon>
         </Compile>
-        <PRIResource Include="**\*.resw" />
+        <PRIResource Include="**\*.resw" Exclude="bin\**\*.resw;obj\**\*.resw" />
```

This is a more general solution that will also work if someone decided to use `UseArtifactsOutput` feature.

```suggestion
        <PRIResource Include="**\*.resw" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
```